### PR TITLE
BS-12991 Do not use a regexp to replace path in server.xml

### PR DESCRIPTION
**Pull request by rbioteau**
Do not use a regexp to replace path in server.xml

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java [hunk 1]
```diff
@@ -224,13 +229,18 @@ private void updateRuntimeLocationIfNeeded() {
...
-                final String oldLocaiton = copy.getLocation().toFile()
+                final String oldLocation = copy.getLocation().toFile()
                         .getAbsolutePath();
                 copy.setLocation(Path.fromOSString(tomcatInstanceLocation));
                 final File serverXmlFile = new File(tomcatInstanceLocation, "conf" + File.separatorChar + "server.xml");
-                // for Windows, we need to escape \
-                FileUtil.replaceStringInFile(serverXmlFile, oldLocaiton,
-                        tomcatInstanceLocation.replaceAll("\\\\", "\\\\\\\\"));
+                try {
+                    com.google.common.io.Files.write(
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java]: I don't see the dependance on com.google.io in the MANIFEST.MF. Can you check if you didn't forget to commit this file?

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java]: it's guava

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java]: I don't see guava neither on https://github.com/bonitasoft/bonita-studio/blob/7.0.x/bundles/plugins/org.bonitasoft.studio.engine/META-INF/MANIFEST.MF

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java]: guava is reexported by org.bonitasoft.studio.common

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.engine/src/org/bonitasoft/studio/engine/BOSWebServerManager.java]: ok

**Pull request merged**